src/pages/Barrels/PoolCard.tsx

```diff
@@ -148,6 +163,16 @@ function PoolCard({ token, hToken, bonusToken }: PoolCardProps) {
           <EntryTitle>Total Reward:</EntryTitle>
           <TokenAmountWithoutIcon symbol={token.symbol} amount={hToken.totalReward} decimals={token.decimals} />
         </Entry>
+
+        <Entry>
+          <EntryTitle>
+            <div style={{ display: 'flex' }}>
+              <span style={{ paddingRight: 5 }}>APY / 1 token</span>
```

Comment:
I think we can just say `Estimated APY`, and add more description in the Help component: 
```
Estimated by being the last participant that stay until the end. This value might change as more participants enter the pool
```